.travis.yml

```diff
@@ -3,8 +3,8 @@ language: node_js
 sudo: false
 
 node_js:
+  - "6"
   - "8"
-  - "10"
```

Comment:
What is the reason for not testing on node v10? locally it seems to work for me, so seems reasonable to keep this so we test on node v10.

Reply:
Good call! I changed it to test Node v10 again now.